bookmarklet.js

```diff
@@ -2677,11 +2677,11 @@ SherdBookmarklet = {
                 asset.sources.poster;
             var newAsset;
             if (img) {
-                newAsset = self.elt(null,'img','sherd-image',{src:img,style:'max-width:215px;max-height:150px',height:null});
```

Comment:
Oh yeah.. makes sense that paul was seeing the overflowing image height problem (PMT #103629) only occurring in the extension. I'll figure this out..